docs/modules/ROOT/pages/type-definitions/schema-configuration/index.adoc

```diff
@@ -0,0 +1,37 @@
+[[type-definitions-schema-configuration]]
```

Comment:
I have the feeling that this page should have the content of type-definitions/schema-configuration.adoc instead. Also, this page is not listed in the content-nav.adoc, which makes me think there was some confusion here as the content is more or less the same.

Reply:
The other way around, while playing with the folder structure I forget to delete the previous attempt which is the file type-definitions/schema-configuration.adoc, my bad!